### PR TITLE
[M] implement watcher for error

### PR DESCRIPTION
**Pull request by Bemhreth**

### CODE REVIEW

**Comment by Bemhreth** [comment]
**test results**
*result of ScreenLogBBRsiScalp_1.91.txt =* https://t.me/c/1658942980/10215
*result of ScreenLogBBRsiScalp_1.90.txt =* https://t.me/c/1658942980/10218
*result of ScreenLogBBRsiScalp_1.86.txt =* https://t.me/c/1658942980/10219

File: wao/_error_watcher.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import subprocess
+import watchdog.events
+import watchdog.observers
+
+from wao.brain_config import BrainConfig
+
+
+class _Error_Watcher(watchdog.events.PatternMatchingEventHandler):
```
Comment (tef-github) [inline wao/_error_watcher.py]: rename to Error_Watcher and error_watcher.py

File: wao/brain_config.py [hunk 2]
```diff
@@ -10,6 +10,7 @@ class BrainConfig:
     EXECUTION_PATH = ROOT + WORKSPACE + "/execution"
     FREQTRADE_PATH = ROOT + WORKSPACE + "/freqtrade"
     _429_DIRECTORY = FREQTRADE_PATH + "/_429_directory/"
+    _FREQTRADE_LOGS_DIRECTORY = FREQTRADE_PATH + "/logs/"
```
Comment (tef-github) [inline wao/brain_config.py]: _WAO_LOGS_DIRECTORY

**Review by tef-github — CHANGES_REQUESTED** [review]
changes requested

File: wao/brain_util.py [hunk 3]
```diff
@@ -57,10 +58,28 @@ def perform_create_429_watcher():
...
+    try:
+        while True:
+            # Set the thread sleep time
+            time.sleep(1)
+    except KeyboardInterrupt:
+        observer.stop()
+    observer.join()
+
+
+def setup():
```
Comment (tef-github) [inline wao/brain_util.py]: create_watchers

**Review by tef-github — CHANGES_REQUESTED** [review]
CR

File: wao/brain_config.py [hunk 4]
```diff
@@ -4,12 +4,14 @@ class BrainConfig:
     BACKTEST_DATA_CLEANER_YEAR = 2021
     BACKTEST_DATA_CLEANER_MONTH_INDEX = 4
     IS_BACKTEST = False
+    IS_ERROR_WATCHER_ENABLED = False
```
Comment (tef-github) [inline wao/brain_config.py:7]: I will set this to true after merging

**Review by tef-github — APPROVED** [review]

**Pull request merged**